src/libraries/System.Security.Cryptography.Cng/src/Internal/Cryptography/BasicSymmetricCipherNCrypt.cs

```diff
@@ -21,7 +21,7 @@ internal sealed class BasicSymmetricCipherNCrypt : BasicSymmetricCipher
         //
         // The delegate must instantiate a new CngKey, based on a new underlying NCryptKeyHandle, each time is called.
         //
-        public BasicSymmetricCipherNCrypt(Func<CngKey> cngKeyFactory, CipherMode cipherMode, int blockSizeInBytes, byte[]? iv, bool encrypting, int feedbackSizeInBytes, int paddingSize)
```

Comment:
We can't do anything with the feedback size here, so stop accepting it so new callers don't assume it does anything.